scripts/taginfo_template.json

```diff
@@ -298,6 +298,20 @@
       "object_types": ["way"],
       "description": "Displays railroads with a thinner line width but normal tie spacing.",
       "doc_url": "https://openmaptiles.org/schema/#transportation"
+    },
+    {
+      "key": "natural",
+      "value": "coastline",
+      "object_types": ["way"],
+      "description": "Coastlines are represented by a hairline at all zoom levels, except across the mouth of a river.",
+      "doc_url": "https://openmaptiles.org/schema/#water"
+    },
```

Comment:
I don't believe that we actually consume OSM coastline tagging, but I might be wrong on that.

Reply:
I believe it's natural earth <z6 and water-polygons ≥z6, the latter being an indirect use of coastline.

Reply:
:+1: that works for me.